tools/gke-disk-image-builder/README.md

```diff
@@ -1,5 +1,5 @@
 # GKE Disk Image Builder
-**[github.com/GoogleCloudPlatform/ai-on-gke/gke-disk-image-builder](https://github.com/GoogleCloudPlatform/ai-on-gke/gke-disk-image-builder)**
+**[github.com/GoogleCloudPlatform/ai-on-gke/tools/gke-disk-image-builder](https://github.com/GoogleCloudPlatform/ai-on-gke/tools/gke-disk-image-builder)**
```

Comment:
This link still 404s: https://github.com/GoogleCloudPlatform/ai-on-gke/tools/gke-disk-image-builder

Reply:
Oh, duh, it's because you're changing the path. Sorry :)